modules/remotecontrol/Controller.js

```diff
@@ -135,6 +137,8 @@ export default class Controller extends RemoteControlParticipant {
             switch(remoteControlEvent.action) {
                 case PERMISSIONS_ACTIONS.grant: {
                     this.controlledParticipant = userId;
+                    logger.debug("Remote control permissions granted to: "
```

Comment:
Same as the comment above, will it make sense to have it logged on the "log" level ?